modules/debug.py

```diff
@@ -70,7 +70,7 @@ def logfile(all=False):
 
 def version():
     title = 'Running version'
-    lines = subprocess.check_output('git log HEAD~1..HEAD ; echo "" ; git status', shell=True)
+    lines = subprocess.check_output('git log HEAD~1..HEAD ; echo "" ; git status', shell=True).decode("utf-8")
```

Comment:
Prefer debug.subprocess_check_output instead

Reply:
Yes, NP

Reply:
OK, I went back a tried these two again, and `debug.subprocess_check_output` does not like the `shell=True` being passed to it.   I think my reasoning the other day was that adding the .decode("utf-8") was fine here because it was the debug module.   I suppose it's also fine to doctor up `subprocess_check_output`  I'll leave it up to you what you prefer.  Here's what I get when I set it as you per your comment:

```
Uh oh, something went wrong...

Please go to github and see if this is a known issue, if not, feel free to file a new issue with the following information:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/root/photoframe/routes/baseroute.py", line 63, in __call__
    ret = self.handle(self.app, **kwargs)
  File "/root/photoframe/routes/debug.py", line 32, in handle
    report.append(debug.version())
  File "/root/photoframe/modules/debug.py", line 73, in version
    lines = subprocess_check_output('git log HEAD~1..HEAD ; echo "" ; git status', shell=True)
TypeError: subprocess_check_output() got an unexpected keyword argument 'shell'
Thank you for your patience
```

Reply:
Yeah, you're right. I thought I had done a `**kwargs` as input to "fix that" but I guess not :( You should retain your original code but please put a comment saying it should use the debug method once it supports shell ... or if you're bored, add kwargs support to debug so it will pass it on automatically. Up to you :)